src/Discord/Parts/Part.php

```diff
@@ -178,12 +178,28 @@ public function fetch(): ExtendedPromiseInterface
     public function fill(array $attributes): void
     {
         foreach ($attributes as $key => $value) {
-            if (in_array($key, $this->fillable)) {
+            if (in_array($key, $this::getFillableAttributes())) {
                 $this->setAttribute($key, $value);
             }
         }
     }
 
+	/**
+     * Returns the fillable attributes.
+     *
+     * @return array
+     */
+    public static function getFillableAttributes($context = '')
```

Comment:
I don't think the function should do any filtering, perhaps just return the `fillable` array and then it can be filtered later.